packages/esm-outpatient-app/src/patient-search/search-results.component.tsx

```diff
@@ -92,8 +92,8 @@ const SearchResults: React.FC<SearchResultsProps> = ({ patients }) => {
           onChange={({ selectedItem }) => setSelectedSortingCriteria(selectedItem.id as SortingCriteria)}
         />
       </div>
-      {sortedPatient.map((patient) => (
-        <div key={patient.id} className={styles.patientChart}>
+      {sortedPatient.map((patient, index) => (
+        <div key={index} className={styles.patientChart}>
```

Comment:
This key is likely to collide. Let's give it a prefix like ``search-result-${i}``

Reply:
```suggestion
        <div key={`search-result-${index}`} className={styles.patientChart}>
```